.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,128 @@
+name: Bug Report
+description: 'You have noticed a general issue or regression, that you would like to report'
+projects:
+  - jellyfin/jellyfin-web
+labels:
+  - bug
+body:
+  - type: textarea
+    id: bug_description
+    validations:
+      required: true
+    attributes:
+      label: Describe The Bug
+      description: A clear and concise description of what the bug is.
+
+  - type: textarea
+    id: reproduction
+    validations:
+      required: true
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+
+  - type: textarea
+    id: expectations
+    validations:
+      required: false
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+
+  - type: textarea
+    id: logs
+    validations:
+      required: false
+    attributes:
+      label: Logs
+      description: Please paste any log errors.
+      placeholder: |
+        You can find the logs at...
+
+      render: bash
+
+
+  - type: textarea
+    id: screenshot
+    validations:
+      required: false
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+
+  - type: dropdown
+    id: systemOS
+    validations:
+      required: true
+    attributes:
+      label: Your OS
+      description: please complete the following information
+      multiple: true
+      options:
+        - Linux
+        - Unix
+        - macOS
+        - Windows
+        - IOS
+        - Android
+        - Tizen
+        - Other (Write which above)
+
+  - type: dropdown
+    id: browser
+    validations:
+      required: true
+    attributes:
+      label: Browser
+      description: Which browser(s) did you use
+      multiple: true
+      options:
+        - Firefox-ESR
+        - Firefox
+        - Chrome based (Chromium, Edge)
```

Comment:
Chromium based.
Chrome is based on Chromium. Edge is based on Chromium.
https://en.wikipedia.org/wiki/Chromium_(web_browser)

Words mixed up.

Reply:
Super, will fix the wording.